packages/core/src/utils/did.ts

```diff
@@ -139,3 +139,14 @@ export function isDid(did: string): boolean {
 export function isDidIdentifier(identifier: string): boolean {
   return DID_IDENTIFIER_REGEX.test(identifier)
 }
+
+/**
+ * Convert publicKeyBase58 to indy did
+ * @param publicKeyBase58
+ * @returns
+ */
+export function indyDidFromPublicKeyBase58(publicKeyBase58: string): string {
```

Comment:
Same here, can you add some tests for this?

Reply:
Test cases added